pkg/cmd/codespace/create.go

```diff
@@ -78,11 +79,16 @@ func newCreateCmd(app *App) *cobra.Command {
 		Use:   "create",
 		Short: "Create a codespace",
 		Args:  noArgsConstraint,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return cmdutil.MutuallyExclusive("using --web with --display-name, --idle-timeout, or --retention-period is not supported", opts.useWeb, opts.displayName != "" || opts.idleTimeout != 0 || opts.retentionPeriod.Duration != nil)
```

Comment:
Nit: break up this long line

```suggestion
            return cmdutil.MutuallyExclusive(
                "using --web with --display-name, --idle-timeout, or --retention-period is not supported",
                opts.useWeb, 
                opts.displayName != "" || opts.idleTimeout != 0 || opts.retentionPeriod.Duration != nil,
            )
```